.github/workflows/pytest.yml

```diff
@@ -47,10 +47,10 @@ jobs:
             ${{ runner.os }}-
 
       - name: Install dependencies
-        run: pip install -r backend/requirements.txt -r backend/requirements-dev.txt codecov
+        run: pip install -r backend/kesaseteli/requirements.txt -r backend/kesaseteli/requirements-dev.txt codecov
```

Comment:
I think we should limit this workflow to only run when there are changes in kesaseteli folder. In helsinki-benefit we use something like this: https://github.com/City-of-Helsinki/yjdh/pull/46/files#diff-280994542061213299f64fa399afda4ec4a787875fe7953d3595900eb09cbc40R6